Intersect.Client/Interface/Game/AdminWindow.cs

```diff
@@ -400,44 +327,34 @@ void _muteButton_Clicked(Base sender, ClickedEventArgs arguments)
         {
             if (TextboxName.Text.Trim().Length > 0)
             {
-                mBanMuteWindow = new BanMuteBox(
-                    Strings.Admin.mutecaption.ToString(TextboxName.Text),
-                    Strings.Admin.muteprompt.ToString(TextboxName.Text), true, MuteUser
-                );
+                mBanMuteWindow = new BanMuteBox(Strings.Admin.mutecaption.ToString(TextboxName.Text),
+                    Strings.Admin.muteprompt.ToString(TextboxName.Text), true, MuteUser);
             }
         }
 
         void MuteUser(object sender, EventArgs e)
         {
-            PacketSender.SendAdminAction(
-                new MuteAction(
-                    TextboxName.Text, mBanMuteWindow.GetDuration(), mBanMuteWindow.GetReason(), mBanMuteWindow.BanIp()
-                )
-            );
+            PacketSender.SendAdminAction(new MuteAction(TextboxName.Text, mBanMuteWindow.GetDuration(),
+                mBanMuteWindow.GetReason(), mBanMuteWindow.BanIp()));
 
             mBanMuteWindow.Dispose();
         }
 
         void BanUser(object sender, EventArgs e)
         {
-            PacketSender.SendAdminAction(
-                new BanAction(
-                    TextboxName.Text, mBanMuteWindow.GetDuration(), mBanMuteWindow.GetReason(), mBanMuteWindow.BanIp()
-                )
-            );
+            PacketSender.SendAdminAction(new BanAction(TextboxName.Text, mBanMuteWindow.GetDuration(),
+                mBanMuteWindow.GetReason(), mBanMuteWindow.BanIp()));
 
             mBanMuteWindow.Dispose();
         }
 
         void _banButton_Clicked(Base sender, ClickedEventArgs arguments)
         {
-            if (TextboxName.Text.Trim().Length > 0 &&
-                TextboxName.Text.Trim().ToLower() != Globals.Me.Name.Trim().ToLower())
+            if (TextboxName.Text.Trim().Length > 0 && !String.Equals(TextboxName.Text.Trim(), Globals.Me.Name.Trim(),
```

Comment:
This is re-trimming a string, that's bad.

Use a IsNullOrWhiteSpace() => return, then trim in the final comparison.

Also, use `string` not `String`. I'm surprised the IDE didn't complain about that?